generateEntityFromLeesStore.py

```diff
@@ -266,9 +266,6 @@ def add_line(path,marker,line,emptyspace):
 copy_tree(fromDirectory, toDirectory)
 
 
-# rename files
-rename_files(new_model,toDirectory)
```

Comment:
This produced an error for me about the destination being the same as the source.  So I deleted it and it worked?  But then I ended up with files like `/scr/app/figs/create-product-create-product-dialog.component.html` rather than `/scr/app/figs/create-fig-create-fig-dialog.component.html` so maybe it needs something else.  lmk and I'll fix it.